stemcell_builder/stages/image_install_grub/apply.sh

```diff
@@ -156,5 +170,12 @@ else
   exit 2
 fi
 
+md5_password=`run_in_chroot ${image_mount_point} "if which grub-md5-crypt > /dev/null;then echo -e '${random_password}\n${random_password}' | grub-md5-crypt | grep -e '^\\$1\\$.*'; fi"`
```

Comment:
@mariash, 
(1). grub-md5-crypt is there when grub V1 is installed. in CentOS 7 system, we only install grub V2, which will use another method(edit the 00_header config file), so in CentOS 7, grub-md5-cyprt is not there.

(2). I don't catch you on the comment, would you please make some detail description? thanks

(3). for system installed grub V2, the grub config file is /boot/grub2/grub.cfg not /boot/grub/grub.conf, so in this case, we don't chmod/chown grub.conf